component/http/component.go

```diff
@@ -124,6 +137,9 @@ type Builder struct {
 // NewBuilder initiates the HTTP component builder chain.
 // The builder instantiates the component using default values for
 // HTTP Port, Alive/Ready check functions and Read/Write timeouts.
+//
+// Deprecated: Pls use the new v2 package.
```

Comment:
Can we do a search&replace `Pls use` -> `Please use`? WDYT?

Reply:
Done